app/models/dialog.rb

```diff
@@ -57,7 +57,7 @@ def automate_values_hash
 
   def validate_children
     # To remove the meaningless error message like "Dialog tabs is invalid" when child's validation fails
-    errors[:dialog_tabs].delete("is invalid")
```

Comment:
As an aside, can we do `errors.delete(:dialog_tabs)`?  It seems to be still valid: https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-delete

Reply:
I agree, I wonder if this was still necessary with rails 6, nevermind 6.1.

Reply:
Looks like this is available in rails 6, so this should be good